modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectValidationRuleUtil.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.admin.rest.internal.dto.v1_0.util;
+
+import com.liferay.object.constants.ObjectValidationRuleSettingConstants;
+import com.liferay.object.model.ObjectField;
+import com.liferay.object.model.ObjectValidationRuleSetting;
+import com.liferay.object.service.ObjectFieldLocalService;
+import com.liferay.object.service.ObjectValidationRuleSettingLocalService;
+import com.liferay.petra.function.transform.TransformUtil;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.List;
+
+/**
+ * @author Pedro Leite
+ */
+public class ObjectValidationRuleUtil {
```

Comment:
If we start using `ObjectValidationRuleResource` in `ObjectDefinitionResourceImpl`, it won't be necessary to isolate this method, right?

If so, undo this modification, please